sw/airborne/subsystems/imu/imu_bebop.c

```diff
@@ -87,10 +93,13 @@ void imu_bebop_init(void)
   //the magnetometer of the bebop2 is located on the gps board,
   //which is under a slight angle
   struct FloatEulers imu_to_mag_eulers =
-          {0.0, RadOfDeg(8.5), 0.0};
+  { 0.0, RadOfDeg( 8.5 ), M_PI };
```

Comment:
This way all relevant magnetometer rotations are grouped in a single FloatEulers structure, in stead of the 180 degree rotation being done on Line 150